### ISSUE TITLE
[Bug]: Rest API put standard ACL

### ISSUE THREAD

**Issue by lvangink** [netbox-community/netbox-acls#190]
### NetBox access-list plugin version

v1.5.0

### NetBox version

v3.7.5

### Steps to Reproduce

When trying to use rest API to add standard access list, the API call ignores the "action" field and will not let me put a new ACL with the source-prefix set.

Example Payload:
`[
    {
    "id": 3,
    "access_list": {
        "id": 7,
        "name": "ACL_PIM_NEIGHBORS"
    },
    "index": 2,
    "action": "permit",
    "source_prefix": {
        "prefix": "0.0.0.3/32"
    }
}]`

Sending without the prefix seems to work:

`[
    {
    "id": 3,
    "access_list": {
        "id": 7,
        "name": "ACL_PIM_NEIGHBORS"
    },
    "index": 2,
    "action": "permit"

}]`

### Expected Behavior

standard ACL rule appears in netbox.

### Observed Behavior

`{
    "source_prefix": [
        "Action is set to remark, Source Prefix CANNOT be set."
    ]
}`

This occurs whether using postman or directly within the swagger interface.

**Comment by cs-1** [comment]
I can confirm that we have the exact behaviour for extended rules:

Sending

```json
{
  "access_list": 1,
  "index": 10,
  "action": "permit",
  "destination_prefix": 2990,
  "source_prefix": 2990
}
```
as POST to `/api/plugins/access-lists/extended-acl-rules/` yields

```json
{
  "source_prefix": [
    "Action is set to remark, Source Prefix CANNOT be set."
  ],
  "destination_prefix": [
    "Action is set to remark, Destination Prefix CANNOT be set."
  ]
}
```

Removing src and dst prefix so that payload looks like this

```json
{
  "access_list": 1,
  "index": 10,
  "action": "permit"
}
```

works. Looking at the code I think that, as soon as you use a source_prefix or destination_prefix, something goes wrong with evaluating the action string in

https://github.com/netbox-community/netbox-acls/blob/1cd9f04c7ba955b9e164bf4bad30ad82b09b2198/netbox_acls/api/serializers.py#L224-L231

Is it possible that the if statement checking the prefix is not indented as it should be?

**Comment by einhirn** [comment]
I've got a proposed fix in https://github.com/einhirn/netbox-acls/commit/b636d24ebfc1706feef9c4661d9b0034d18d7ac7 but I didn't want to create an untested PR. Will test tomorrow and create PR if it works as expected.

**Comment by cs-1** [comment]
> I've got a proposed fix in [einhirn@b636d24](https://github.com/einhirn/netbox-acls/commit/b636d24ebfc1706feef9c4661d9b0034d18d7ac7) but I didn't want to create an untested PR. Will test tomorrow and create PR if it works as expected.

Your patch works. I've just imported ~180 ACLs with a total of ~1800 ACEs and all worked as expected.
